pkg/common/resource_utils/autoscaler.go

```diff
@@ -48,7 +48,7 @@ func BuildHorizontalPodAutoscaler(pap *PodAutoscalerParams) client.Object {
 	switch pap.AutoscalerType {
 	case srapi.AutoScalerV1:
 		return buildAutoscalerV1(pap)
-	case srapi.AutoSclaerV2:
+	case srapi.AutoScalerV2:
```

Comment:
looks like this code path is never touched?

Reply:
If `starrocksCnSpec.autoScalingPolicy.version` is set v2, this code path will be executed.